state-chain/custom-rpc/src/lib.rs

```diff
@@ -451,6 +465,12 @@ pub trait CustomApi {
 	#[subscription(name = "subscribe_prewitness_swaps", item = BlockUpdate<RpcPrewitnessedSwap>)]
 	fn cf_subscribe_prewitness_swaps(&self, base_asset: Asset, quote_asset: Asset, side: Order);
 
+	// Subscribe to a stream that on every block produces a list of all scheduled/pending
+	// swaps in the base_asset/quote_asset pool, including any "implicit" half-swaps (as a
+	// part of a swap involving two pools)
+	#[subscription(name = "subscribe_scheduled_swaps", item = SwapResponse)]
+	fn cf_subscribe_scheduled_swaps(&self, base_asset: Asset, quote_asset: Asset);
```

Comment:
I suggest we add a cf_scheduled_swaps(base_asset, quote_asset, at) rpc too

Reply:
Was a request of LPs, helps them now what as happened in the past

Reply:
added